File: Data Structures & Algorithms/majority-element/submission-0.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        unordered_map<int,int>mp;
        int max=0;
        int res=nums[0];
        for(int i=0;i<nums.size();i++){
            mp[nums[i]]++;
            if(mp[nums[i]]>max){
                max=mp[nums[i]];
                res=nums[i];
            }
        }
        return res;
    }
};

File: Data Structures & Algorithms/sliding-window-maximum/submission-0.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        res = []
        res.append(max(nums[0:k]))

        l = 1
        r = k

        while r < len(nums):
            res.append(max(nums[l:r+1])) 
            l += 1
            r += 1

        return res
            
